lunatrace/dev-cli/src/services.ts

```diff
@@ -62,3 +62,13 @@ export const sbomWorker = tmuxPane([
 export const smeeWebhook = tmuxPane([
   `smee -u ${smeeWebhookUrl} -p 3002 -P /github/webhook/events`
 ])
+
+export const generateCommon = tmuxPane([
+  `cd common`,
+  `yarn start`
+])
+
+export const generateLogger = tmuxPane([
+  `cd logger`,
```

Comment:
Nice.  This is what I was thinking too but we should probably do what free suggested with the tsconfig.build.json.  heres the one from lunadefend
```
{
  "files": [],
  "references": [
    { "path": "packages/isomorphic-common" },
    { "path": "packages/server-common"},
    { "path": "packages/node-sdk" },
    { "path": "packages/tokenizer-sdk" },
    { "path": "packages/browser-common" },
    { "path": "packages/react-sdk" },
    { "path": "packages/cli" },
    { "path": "packages/secure-frame-iframe" },
    { "path": "packages/vue-sdk" },
    { "path": "packages/server-common/tsconfig.module.json"},
    { "path": "packages/browser-common/tsconfig.module.json"},
    { "path": "packages/node-sdk/tsconfig.module.json" },
    { "path": "packages/tokenizer-sdk/tsconfig.module.json" },
    { "path": "packages/react-sdk/tsconfig.module.json" },
    { "path": "packages/isomorphic-common/tsconfig.module.json" },

  ]
}
```
honestly though, this works too, so whatever